applications/asset_tracker_v2/src/events/app_module_event.h

```diff
@@ -47,6 +47,16 @@ enum app_module_event_type {
 	/** Request latest configuration from the cloud. */
 	APP_EVT_CONFIG_GET,
 
+	/** Application module is waiting for movement to trigger the next sample request. This
+	 * event is used to signal the sensor module to enable activity detection.
```

Comment:
Nit: `event` should align with `Application`. Same for enum below

Reply:
Yes. Ill update the rest of the doc for this file as well.